lib/testcase.rb

```diff
@@ -731,7 +732,10 @@ def get_default_log_check_levels
     :log4j_initialize => /log4j:WARN Please initialize the log4j system properly/,
     :log4j_see_noconfig => /log4j:WARN See http:\/\/logging.apache.org\/log4j\/1.2\/faq.html#noconfig/,
     :concurrent_mark_sweep => /OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated in version 9.0 and will likely be removed in a future release./,
-    :zookeeper_shutdown => /Starting non-reconfigurable ZooKeeper server failed on attempt/
+    :zookeeper_shutdown => /Starting non-reconfigurable ZooKeeper server failed on attempt/,
+    :time_move_backwards => /Time has moved backwards/,
```

Comment:
Did you intend to add `:time_move_backwards` to the list of ignorable messages as well?

Reply:
No, I just wanted in the list of possible warnings. Time does move backwards from time to time with docker-on-mac.
This just makes it a bit easier ignore it when running locally.
It should not be ignored normally.

Reply:
👍